spec/features/user_packs_spec.rb

```diff
@@ -0,0 +1,61 @@
+require 'rails_helper'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.